conda_build/environ.py

```diff
@@ -545,12 +543,11 @@ def meta_vars(meta, skip_build_id=False):
     ):
         d.update(get_hg_build_info(hg_dir))
 
-    # use `get_value` to prevent early exit while name is still unresolved during rendering
-    d["PKG_NAME"] = meta.get_value("package/name")
+    d["PKG_NAME"] = meta.name()
```

Comment:
it's almost surprising these are not properties `meta.name`

Reply:
yup, especially surprising when other values of `MetaData` are properties 🤷🏻‍♂️